README.md

```diff
@@ -161,7 +161,7 @@ To test, you will use a Node.js script (`bin/msg.mjs`). Follow these steps:
 
    `export ID_TOKEN="<the ID token from step 2>"`
 
-   `node bin/msg.mjs send-message "Serverless is cool\!" -w 20`
+   `node bin/msg.mjs send-message "Serverless is cool\!" -w 20 --region <region_where_stack_is_deployed>`
```

Comment:
About the "--region", either we add the parameter in all instructions or we put a instruction to create a environment variable "AWS_DEFAULT_REGION". 

To make it simple, I'd go to create the environment variable.

Reply:
Me too, I'll add it